cmd/cli/kubectl-kyverno/test/load.go

```diff
@@ -73,6 +74,15 @@ func LoadTest(fs billy.Filesystem, path string) TestCase {
 			Err:  err,
 		}
 	}
+
+	if test.APIVersion != "cli.kyverno.io/v1alpha1" || test.Kind != "Test" {
+		return TestCase{
+			Path: path,
+			Fs:   fs,
+			Err:  fmt.Errorf("invalid or missing schema: test must include apiVersion: cli.kyverno.io/v1alpha1 and kind: Test"),
+		}
+	}
+
```

Comment:
No need for this change as you already added a check [here](https://github.com/kyverno/kyverno/blob/7fefb53963e916c09ff44c9235ac3673b7d7dcd7/cmd/cli/kubectl-kyverno/variables/new.go#L21).